### ISSUE TITLE
Bottom-up CrowdPose demo error

### ISSUE THREAD

**Issue by carollowbee** [open-mmlab/mmpose#646]
Hi! I got an error when i run like this:
python demo/bottom_up_video_demo.py configs/bottom_up/higherhrnet/crowdpose/higher_hrnet32_crowdpose_512x512.py checkpoints/higher_hrnet32_crowdpose_512x512-1aa4a132_20201017.pth --video-path data/crowdpose/videos/1.mp4 --out-video-root vis_results

the error is like that:
RuntimeError: CUDA error: device-side assert triggered
terminate called after throwing an instance of 'c10::Error'
  what():  CUDA error: device-side assert triggered

But when I run with  coco py and pth, it's good.
So where's the problem, can you help me?

**Comment by innerlee** [comment]
The error is likely to be caused by out of range class indices.

**Comment by innerlee** [comment]
``` bash
$ python demo/bottom_up_video_demo.py configs/bottom_up/higherhrnet/crowdpose/higher_hrnet32_crowdpose_512x512.py https://download.openmmlab.com/mmpose/bottom_up/higher_hrnet32_crowdpose_512x512-1aa4a132_20201017.pth --video-path data/crowdpose/videos/1.mp4 --out-video-root vis_results
Use load_from_http loader
Traceback (most recent call last):
  File "demo/bottom_up_video_demo.py", line 108, in <module>
    main()
  File "demo/bottom_up_video_demo.py", line 45, in main
    assert (dataset == 'BottomUpCocoDataset')
AssertionError
```

Did you modified the code?

**Comment by carollowbee** [comment]
yes,I just modified it here and there is no modification elsewhere.I didn't modify the code and model, I used the code you provided directly, so I don't know how to revision the error class indices

**Comment by innerlee** [comment]
@jin-s13 I guess we haven't supportted the bot-up crowd-pose demo?

**Comment by carollowbee** [comment]
Well, I'm looking forward to your release

**Comment by jin-s13** [comment]
This will be fixed in #663
